### DISCUSSION TITLE
Does K2 support forced alignment like kaldi?

### DISCUSSION THREAD

**Discussion by fightingtang** [k2-fsa/sherpa#383] [Q&A]
Excuse me, I was wondering if you happen to know whether k2 supports forced alignment or not? Thank you for your time.

**Comment by trunglebka** [comment]
This is what I want to know too.
I think that we can do it with CTC recipes but may need to do it ourselves since icefall (it would be implemented there if any - I think) is far from mature like Kaldi and that document and support for unpopular use case may be lack.
